src/components/05_pages/Permissions/Permissions.jsx

```diff
@@ -1,12 +1,63 @@
-import React from 'react';
+import React, { Fragment, Component } from 'react';
 import { css } from 'emotion';
 
-const Permissions = () => (
-  <div>
-    <h1 className={styles.title}>Permissions</h1>
-    <p>This will be the permissions page.</p>
-  </div>
-);
+const drupalOrigin = 'http://localhost:9998';
+
+const Permissions = class Permissions extends Component {
+  state = {
+    loaded: false,
+  };
+  componentDidMount() {
+    fetch(`${drupalOrigin}/permissions?_format=json`)
+      .then(res => res.json())
+      .then(permissions => {
+        const groupedPermissions = {};
+        Object.keys(permissions).forEach(key => {
+          groupedPermissions[permissions[key].provider] =
+            groupedPermissions[permissions[key].provider] || [];
+          groupedPermissions[permissions[key].provider].push(permissions[key]);
+        });
+        return { groupedPermissions, loaded: true };
+      })
+      .then(data => this.setState({ ...data }))
+      .catch(console.log);
```

Comment:
Let's console.error that?

Reply:
Maybe we could route it through winston and have `logger.error` here instead?
https://www.npmjs.com/package/winston

Reply:
Should we create a quick ticket to track this? For me ideally we write the data one day back to Drupal